### ISSUE TITLE
Determine goals for the advisory group

### ISSUE THREAD

**Issue by libcce** [LibraryCarpentry/governance#13]
In our Advisory Group meeting on 11 June 2019, we gave ourselves a homework assignment, to think about our goals for the next year/two years. The previous goals can be found here:

https://librarycarpentry.org/advisory/

Here are the following themes/goals that we discussed as possibilities:

- Map Data, Software, Library Carpentry curriculum/lessons to job descriptions/accreditation/quality standards (list standards at the national level)
- Your local/regional goal that can help at the community-wide level
- Building sustainability into Library, Software, Data Carpentry (report to management/letter of interest)
- Communication/outreach to librarians, helping them understand Library Carpentry/The Carpentries

@davidfkane @goldmj @elibtronic @ragamouf @Jaredabera @konrad @gitti1 @arieldeardorff @jcoliver @chodacki please provide feedback regarding this list by 21 June. After we've put together a list, we can vote and prioritize.

**Comment by jcoliver** [comment]
+ Another way to map Carpentries is via competencies. Many of the individual lessons' learning objectives provide suggestions, if not outright competencies themselves. Having language that participants could use on a CV/Resume could be useful.
+ Local goals are great!
+ I'm interested to hear more about sustainability. Other than hiring people and putting Carpentries in their job description (not that that is a bad thing!), what other efforts can be made for resilient support of Carpentries?
+ I think there is growing appreciation for Carpentries in libraries; getting folks on board early in their career (i.e. during MLS/MLIS) is a great way for long-term engagement. To reach folks more established in their careers, we have had success leading with the way LC can make their lives easier with the Tidy Data and OpenRefine lessons. The SQL lesson gets a lot of traction, too, because, you know, searching.

**Comment by libcce** [comment]
Via Gitter https://gitter.im/LibraryCarpentry/Lobby @jyssy asked if we have a close connection with library associations, specifically ALA. Most of our interactions have been conversations that have led to presentations, workshops, birds of a feathers... There was also a recent suggestion whether we could set up a booth at the Public Library Association Meeting. Library Carpentry has grown a great deal and I've been wondering whether we are ready for a LibraryCarpentryCon. I think it would help if we had a chance to meet in person.

**Comment by chodacki** [comment]
I like the idea of targeted community outreach to librarians at existing Carpentries members.  Building more engagement on those campuses (within the library).  Ensuring all Carpentries members see the opportunities associated with staff embracing LC, retooling librarians, bridging groups on their campuses, etc.

**Comment by arieldeardorff** [comment]
I am also really focused on the idea of outreach as I think there is a lot we can do in this area. Maybe a goal for this year could be to come up with a list of library organizations/conferences/groups that we want to reach out to and create a basic outreach/marketing presentation that can be modified to fit a variety of audiences (this might already exist but I couldn't find it on the website).

Also, I wonder if we want to describe other ways of teaching the material that don't require hosting an entire workshop. Are there pathways we can create for librarians that want to teach one of the lessons as a standalone session (we actually do this a lot at my library for SWC lessons). Maybe we can draft some ideas for using the materials in other ways (ie: 30 minute coffee break sessions, as a month of brown bag lunch and learns, as a flipped classroom discussion group...). This might also help people figure out how to incorporate the Top 10 Fair Data things into their teaching.

**Comment by katrinleinweber** [comment]
The general Carpentries presentation folder contains some LC files as well: https://drive.google.com/drive/folders/1v1E3QqGKcif7LRPVsjKOxudMKbvYsJ04

About the list idea: I suggest to actually reach out to the library organizations/conferences/groups using templates such as from https://github.com/carpentries/commons. Tracking those contacts afterwards, LC-internally builds the list.

**Comment by goldmj** [comment]
Related to @arieldeardorff's point about using Carpentries teaching materials - I really like how LC makes the distinction between 'days' vs 'parts' when running workshops. It is great to see that "as long as the core lessons are taught, it makes no difference whether the workshop is run in one day, two days, or multiple days.” This is great for series of workshops, say, half a day once a week for 4 weeks can still be advertised as LC. I would be interested to see if this kind of format could be normalized. This is an important conversation for the Carpentries to have to make them more flexible. Maybe we could make this a thing at LC as a proof of concept.

Related to "Communication/outreach to librarians, helping them understand Library Carpentry/The Carpentries" - it would be very helpful to craft use cased (similar to the personas) that show workflows or real examples on how librarians use the Carpentries topics in their daily work. More real cases and actual datasets that can be used within the training materials would get people onboard.

**Comment by ragamouf** [comment]
Mapping curricula to competencies is a good move. Nice one @jcoliver! 

Also, I really like targetting outreach to librarians where there are known carpentries people amongst research communities @chodacki that's a great strategy. We can do good work joining people up.

I'm looking forward to hearing more about people's local initiatives and activities across this Advisory Group.

I wanted to share a couple of local initiatives that I'm involved in here in Sydney. Firstly, I'm coordinating a national community of practice for people involved in Library Carpentry - this is serving as a way of connecting LC instructors with what everyone else is doing, and providing a gentle onboarding for librarians and research support workers (we're getting interest from govt departments and research organisations too) who are interested and want to know more. LC (and carpentries) lends itself readily to addressing local concerns, but it's evident that those involved value hearing other stories of applying and customising lessons in moderately similar timezones. So there's a role in coordinating and keeping people in the loop. 

Secondly, I'm part of the organising committee for the Sydney ResBaz (a 3 day festival of intro to coding workshops for PhD students and Early Career Researchers), which this year is adding a Librarians and Research Support Stream to it's standard offerings of carpentries workshops, festival of research tool lightning talks and THATCamp unconference for humanities scholars. One of the workshops for this Librarians stream is called Walk the Talk: Data for internal library processes, and will showcase examples from LC (among others) where fellow librarians share skills in good data management practice. I see this as a working example of @goldmj's excellent suggestion for more real use cases and persona development. 

In summary - or rather, back to @libcce's original prompt, the 4 themes are sufficient to cover all the excellent directions people have suggested. I'm wary of committing to too many things, so it would help to have more clearly defined expectations surrounding participating in this AG, eg number hours per week or month. Perhaps each theme could have a couple of co-champions who manage it for the rest of the Advisory Group?

**Comment by libcce** [comment]
Advisory Group has voted and the goals are posted https://librarycarpentry.org/advisory/
